### DISCUSSION TITLE
Change velocity NoteOFF

### DISCUSSION THREAD

**Discussion by dsg25** [tttapa/Control-Surface#746] [Q&A] [answered]
Hi! Help, Im novice in Arduino.
How in this code change velocity NoteOff?

#include <Control_Surface.h> 

USBMIDI_Interface midi;

NoteButton button {
  5,                       // Push button on pin 5
  {MIDI_Notes::C(4), CHANNEL_1}, // Note C4 on MIDI channel 1
   0X6e,
};

void setup() {
  Control_Surface.begin(); // Initialize Control Surface
}

void loop() {
  Control_Surface.loop(); // Update the Control Surface
}

**Comment by tttapa** [comment, marked as answer]
The standard NoteButton class doesn't allow you to change the on/off velocities separately.

See https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d3b/Custom-MIDI-Output-Element_8ino-example.html

**Reply by dsg25** [reply]
Thanks!
